atproto/identity/redisdir/promethus.go

```diff
@@ -0,0 +1,36 @@
+package redisdir
```

Comment:
My convention has been to name this file `metrics.go`

Reply:
you didn't like "promethus" :sweat_smile: